vscode/src/copilot/webview/copilot.css

```diff
@@ -0,0 +1,177 @@
+/* Copyright (c) Microsoft Corporation.
+   Licensed under the MIT License. */
+
+body {
+  padding: 12px;
+  height: 100%;
+  /*  match the native vscode chat style */
+  font-family:
+    Segoe WPC,
+    Segoe UI,
+    sans-serif;
+  font-size: 13px;
+  background-color: var(--vscode-chat-list-background);
+}
+
+html {
+  height: 100%;
+}
+
+h1 {
+  /* match the native vscode chat style */
+  font-size: 24px;
+  font-weight: 500;
+  margin-top: 10px;
+}
+
+.welcome-message {
+  text-align: center;
+}
+
+.app {
+  max-width: 800px;
+  display: flex;
+  flex-direction: column;
+  height: 100%;
+}
+
+/* Whole chat container including the input box */
+.chat {
+  flex: 1;
+}
+
+.status-indicator {
+  height: 30px;
+  font-weight: bold;
+  text-align: right;
+  font-size: smaller;
+}
+
+.input-textarea {
+  height: auto;
+  min-height: calc(1.33em * 2);
+  background: inherit;
+  width: 100%;
+  border: none;
+  outline: none;
+  overflow-y: auto;
+  padding: 16px 10px;
+  margin: 0;
+  line-height: 1.33;
+  vertical-align: middle;
+  cursor: text;
+  resize: none;
+  color: var(--vscode-input-foreground);
+  background: var(--vscode-input-background);
+}
+
+.input-textarea:focus {
+  outline: none;
+}
+
+.send-button {
+  padding: 2px;
+}
+
+.input-box-divider {
+  height: 50px;
+}
+
+.right-message-row,
+.left-message-row {
+  display: flex;
+  flex-direction: row;
+}
+
+.right-message-row {
+  justify-content: flex-end;
+}
+
+.left-message-row {
+  justify-content: flex-start;
+}
+
+.user-message,
+.assistant-message {
+  padding: 0 16px;
+  margin-bottom: 16px;
+  border: 1px solid #ccc;
```

Comment:
Some colors (like 'gray') are theme agnostic, but is this very light color correct in both dark and light themes?

Reply:
This looked fine to my eyes in both modes, but I'm making a note to go back and refine the colors after I merge the PR